consensus/ethash/consensus.go

```diff
@@ -624,14 +624,18 @@ var (
 // reward. The total reward consists of the static block reward and rewards for
 // included uncles. The coinbase of each uncle block is also rewarded.
 func accumulateRewards(config *params.ChainConfig, state *state.StateDB, header *types.Header, uncles []*types.Header) {
+
+	//NOTE: this code will be executed for Raft on Quorum, so the static block reward also applies to Raft
+	blockReward := big.NewInt(0)
 	// Select the correct block reward based on chain progression
-	blockReward := FrontierBlockReward
+	blockReward = FrontierBlockReward
```

Comment:
a condition to add here?

Reply:
I had added a condition, but didn't need it in the end. It looks like I left the partial change there - will correct it.

Reply:
Corrected